transit_odp/publish/requires_attention.py

```diff
@@ -0,0 +1,124 @@
+from typing import Dict, List
+
+from django.utils.timezone import now
+
+from transit_odp.organisation.models.data import TXCFileAttributes
+from transit_odp.otc.models import Service as OTCService
+
+
+def get_otc_map(org_id: int) -> Dict[str, OTCService]:
+    """
+    Get a list of dictionaries which includes all OTC Services for an organisation,
+    excluding exempted services and Out of Season seasonal services.
+    """
+    return {
+        service.registration_number.replace("/", ":"): service
+        for service in OTCService.objects.get_otc_data_for_organisation(org_id)
+    }
+
+
+def get_txc_map(org_id: int) -> Dict[str, TXCFileAttributes]:
+    """
+    Get a list of dictionaries of live TXCFileAttributes for an organisation
+    with relevant effective staleness dates annotated.
+    """
+    return {
+        txcfa.service_code: txcfa
+        for txcfa in TXCFileAttributes.objects.filter(
+            revision__dataset__organisation_id=org_id
+        ).add_staleness_dates()
+    }
```

Comment:
How does this return only the live revisions?

Reply:
It's badly named in that case - if a method begins "add", I'm not expecting it to filter rows out. I would prefer that you either renamed the method or called `get_active_live_revisions` explicitly from here (probably the latter).